### PR TITLE
Update Readme and Compatibility

### PR DESCRIPTION
**Pull request by gxalpha**
I couldn't check all apps for compatibility as I don't have them.
Also added build instructions for branch and updated some infos in readme.

### CODE REVIEW

**Comment by gxalpha** [comment]
Should I start a new PR to master here?

**Comment by johnboiles** [comment]
Looks like I can just change the base though we'll need to fix some conflicts.

**Comment by gxalpha** [comment]
should be good now

File: COMPATIBILITY.md [hunk 1]
```diff
@@ -8,12 +8,11 @@ Please note that this list is not complete.
...
 |Zoom|Yes*|
 |Skype|Yes|
-|Microsoft Teams|Yes|
-|Discord|No|
+|Discord|Yes|
 |QuickTime Player|Yes|
-|Photo Booth|No|
-|FaceTime|No|
-|Safari|No (?)|
+|Photo Booth|Yes (4:3 only)|
```
[the remark this answers is not in the document]
Comment (johnboiles) [reply COMPATIBILITY.md:13]: I probably need to fix other resolutions with the new DAL plugin for this to be true.

File: README.md [hunk 2]
```diff
@@ -8,13 +8,12 @@ Creates a virtual webcam device from the output of [OBS Studio](https://obsproje
 
 This code was spun out of [this OBS Project RFC](https://github.com/obsproject/rfcs/pull/15) which was itself spun out of [this issue](https://github.com/obsproject/obs-studio/issues/2568) from [@tobi](https://github.com/tobi). This was intended as a proof of concept to inform technical decisions in that RFC, but who knows, maybe it will be useful to you in its current state. The goal is for this (or something with equivalent functionality) to eventually be merged into the core OBS codebase 🤞.
 
-This is heavily based on [Apple's CoreMediaIO sample code](https://developer.apple.com/library/archive/samplecode/CoreMediaIO/Introduction/Intro.html) which has been [modernized by @lvsti](https://github.com/lvsti/CoreMediaIO-DAL-Example)
+This is heavily based on [Apple's CoreMediaIO sample code](https://developer.apple.com/library/archive/samplecode/CoreMediaIO/Introduction/Intro.html) which has been [modernized by @lvsti](https://github.com/lvsti/CoreMediaIO-DAL-Example) and on [CoreMediaIO Device Abstraction Layer (DAL) Minimal Example by @johnboiles](https://github.com/johnboiles/coremediaio-dal-minimal-example).
```
Comment (johnboiles) [inline README.md:11]: <3

File: README.md [hunk 3]
```diff
@@ -8,13 +8,12 @@ Creates a virtual webcam device from the output of [OBS Studio](https://obsproje
...
 
 ## Known Issues
 
 ### Plugin Bugs
 * The virtual camera doesn't work with all programs. Check out [COMPATIBILITY.md](https://github.com/johnboiles/obs-mac-virtualcam/blob/master/COMPATIBILITY.md) to see if your app is supported.
-* The virtual camera must be started before the program using it is, otherwise it will not show up
```
Comment (johnboiles) [inline README.md:17]: 🎉

File: README.md [hunk 4]
```diff
@@ -61,6 +60,7 @@ cmake .. && make -j
 cd ../..
 git clone https://github.com/johnboiles/obs-mac-virtualcam.git
 cd obs-mac-virtualcam
+git checkout new-dal-plugin
```
Comment (johnboiles) [inline README.md:63]: No longer necessary

File: COMPATIBILITY.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gxalpha) [reply COMPATIBILITY.md:13]: Yes, it does work for me. It just cuts off the borders left and right to make it 4:3 (at least I think it's 4:3, but it definitely cuts something off)

[the remark this answers is not in the document]
Comment (johnboiles) [reply COMPATIBILITY.md:13]: Interesting! I can't seem to get it to work for me.

**Pull request merged**